btree/btree.c

```diff
@@ -3509,11 +3509,17 @@ static int64_t btree_put_kv_tick(struct m0_sm_op *smop)
 			 * other thread which has lock is working on the same
 			 * node(lev->l_node) which is pointed by current thread.
 			 */
-			if (!node_isvalid(lev->l_node)) {
+
+			lev->l_node->n_skip_rec_count_check = true;
+			if (!node_isvalid(lev->l_node) || (oi->i_used > 0 &&
```

Comment:
Is `oi->i_used > 0` check required?

Reply:
Yes, Because at root node, node_rec_count can be 0 if tree is empty , and it is valid node. Therefore, oi->i_used>0 check is needed as we want to do this check for all other nodes except root node.

Reply:
Ok